File: javatests/com/thoughtworks/selenium/corebased/TestClickBlankTarget.java
package com.thoughtworks.selenium.corebased;

import com.thoughtworks.selenium.InternalSelenseTestNgBase;
import org.testng.annotations.Test;

public class TestClickBlankTarget extends InternalSelenseTestNgBase {
	@Test
  public void testClickBlankTarget() throws Exception {
		selenium.open("../tests/html/Frames.html");
		selenium.selectFrame("bottomFrame");
		selenium.click("changeBlank");
		selenium.waitForPopUp("_blank", "10000");
		selenium.selectWindow("_blank");
		selenium.click("changeSpan");
		selenium.close();
		selenium.selectWindow("null");
		selenium.click("changeBlank");
		selenium.waitForPopUp("_blank", "10000");
		selenium.selectWindow("_blank");
		selenium.click("changeSpan");
		selenium.close();
		selenium.selectWindow("null");
		selenium.selectFrame("bottomFrame");
		selenium.submit("formBlank");
		selenium.waitForPopUp("_blank", "10000");
		selenium.selectWindow("_blank");
		selenium.click("changeSpan");
		selenium.close();
		selenium.selectWindow("null");
		selenium.open("../tests/html/test_select_window.html");
		selenium.click("popupBlank");
		selenium.waitForPopUp("_blank", "10000");
		selenium.selectWindow("_blank");
    System.out.println("At the end");
		verifyEquals(selenium.getTitle(), "Select Window Popup");
		selenium.close();
    selenium.selectWindow("null");
	}
}

File: dotnet/test/webdriverbackedselenium.tests/TestClickBlankTarget.cs
using System;
using NUnit.Framework;

namespace Selenium.Tests
{
    [TestFixture]
    public class TestClickBlankTarget : SeleniumTestCaseBase
    {
        [Ignore("Incorrect test")]
        public void ClickBlankTarget()
        {
            selenium.Open("../tests/html/Frames.html");
            selenium.SelectFrame("bottomFrame");
            selenium.Click("changeBlank");
            selenium.WaitForPopUp("_blank", "10000");
            selenium.SelectWindow("_blank");
            selenium.Click("changeSpan");
            selenium.Close();
            selenium.SelectWindow("null");
            selenium.Click("changeBlank");
            selenium.WaitForPopUp("_blank", "10000");
            selenium.SelectWindow("_blank");
            selenium.Click("changeSpan");
            selenium.Close();
            selenium.SelectWindow("null");
            selenium.Submit("formBlank");
            selenium.WaitForPopUp("_blank", "10000");
            selenium.SelectWindow("_blank");
            selenium.Click("changeSpan");
            selenium.Close();
            selenium.SelectWindow("null");
            selenium.Open("../tests/html/test_select_window.html");
            selenium.Click("popupBlank");
            selenium.WaitForPopUp("_blank", "10000");
            selenium.SelectWindow("_blank");
            Console.WriteLine("At the end");
            Assert.AreEqual("Select Window Popup", selenium.GetTitle());
            selenium.Close();
            selenium.SelectWindow("null");
        }
    }
}
